### PR TITLE
Update container agent helm install instructions

### PR DESCRIPTION
**Pull request by Stuart133**
# Description

Clarify helm values file creation

# Reasons

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/container-runner-installation.adoc [hunk 1]
```diff
@@ -32,7 +34,7 @@ agent:
       token: MY_TOKEN
```
Comment (sebastian-lerner) [inline jekyll/_cci2/container-runner-installation.adoc:34]: can we specify maybe in the comments that this is the resource class token

**Review by sebastian-lerner — APPROVED** [review]
LGTM

File: jekyll/_cci2/container-runner-installation.adoc [hunk 2]
```diff
@@ -32,7 +34,7 @@ agent:
...
 ```
 +
-4. Run `helm install container-agent container-agent/container-agent -n circleci`
+5. Run `helm install container-agent container-agent/container-agent -n circleci -f values.yaml`
```
Comment (kelvintaywl) [inline jekyll/_cci2/container-runner-installation.adoc:37]: [sharing; no action needed]

I was also able to "squeeze" step 3 into step 5 via passing the `--create-namespace` flag in step 5 instead

```console
helm install container-agent container-agent/container-agent -n circleci --create-namespace -f values.yaml
```

**Comment by sebastian-lerner** [comment]
actually one thing stuart, "This page is a continuation of installing self-hosted runners. You will need to have an existing namespace and resource class to continue below. You can do this on the [CircleCI web app](https://app.circleci.com/)"

can we clarify in this blurb that this reference to namespace is a CircleCI namespace? to avoid confusion with Kubernetes namespace.

**Review by christian-stephen — APPROVED** [review]

**Review by devinmarieb — APPROVED** [review]

**Comment by devinmarieb** [comment]
Thank you 🙌

**Pull request merged**